src/Eshopworld.Tests.Core.Analyzers/Eshopworld.Tests.Core.Analyzers.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>Eshopworld.Tests.Core.Analyzers</id>
-    <version>1.0.8</version>
+    <version>1.0.9</version>
     <title>Eshopworld.Tests.Core.Analyzers</title>
     <authors>David Rodrigues, Oisin Haken</authors>
     <owners>eShopWorld</owners>
```

Comment:
Can you add a description to the releaseNotes node

Reply:
sure